github/event_test.go

```diff
@@ -30,6 +30,11 @@ func TestPayload_NoPanic(t *testing.T) {
 	e.Payload()
 }
 
+func TestEmptyEvent_NoPanic(t *testing.T) {
+	e := &Event{}
+	e.ParsePayload()
+}
```

Comment:
Also, please:

```suggestion
  var nilEvent *Event
  nilEvent.ParsePayload()
}
```

Reply:
Done.

Reply:
(I also checked that both of these returned error, which seems correct.)